RELEASE_NOTES.md

```diff
@@ -1,5 +1,13 @@
 # Next release
 
+## API changes
+
+PR [#334](https://github.com/IAMconsortium/pyam/pull/334) changes the second input name of `fill_series()` to `time`. It can still be an integer. 
```

Comment:
```suggestion
PR [#334](https://github.com/IAMconsortium/pyam/pull/334) changes the arguments of `IamDataFrame.interpolate()` and `pyam.fill_series()` to `time`. It can still be an integer (i.e., year).
```